### ISSUE TITLE
[FEATURE] RBIN pack function

### ISSUE THREAD

**Issue by Another759** [OpenKH/OpenKh#776]
Hi, I am not sure if  it was addressed before, but would it maybe be possible to add a option to pack rbin files from directories?
I have searched far and wide for a tool that supports this function but sadly all of the older links seem to be broken or removed.

**Comment by Delta-473** [comment]
Is this for the 3DS version of KH DDD?

**Comment by Another759** [comment]
Yes, while there are tools avaible that let you extract rbin files, I wasn't able to find a tool with a working function of repacking folders into rbin. I know that there was atleast one tool which could do this, but after searching for many hours I still wasn't able to find anything. This is pretty much my last hope.

**Comment by Vladabdf** [comment]
We haven't quite explored DDD nearly as much as any other game covered within the scope of OpenKH. As a result, currently our tooling only supports extraction, not repacking. I'm sure it wouldn't be too difficult to just simply reverse the function and get a working rbin repacker functionality in our `OpenKh.Command.Rbin` program though...

**Comment by Another759** [comment]
That would be amazing as it would enable you to modify game content (e.g. changing models). I really hope you can add it when you are not too busy.
